config/routes.rb

```diff
@@ -1,10 +1,9 @@
 Rails.application.routes.draw do
-  mount_devise_token_auth_for 'User', 
-  at: 'auth', 
-  controllers: { omniauth_callbacks: 'api/omniauth_callbacks' }
+  mount_devise_token_auth_for 'User', at: 'auth', controllers: { omniauth_callbacks: 'api/omniauth_callbacks' }
```

Comment:
I would prefer to move this back to the way it was with "at:" and "controllers:" on different lines. just for readability

Reply:
it is the formatter.

Reply:
We put them in different lines though👍